common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx

```diff
@@ -292,8 +292,20 @@ const ViewerTopBar: FunctionComponent<Props> = ({
                     }
                   }}
                 >
-                  <Icon name="expand" />
-                  <span className={`btn__text`}>Full screen</span>
+                  <Icon
```

Comment:
just wondering if wrapping the duplicating the Icon and span so they can be wrapped in the same conditional might be better/easier to read

Reply:
yeah, that's better. Will update.